mitmproxy/builtins/script.py

```diff
@@ -16,6 +16,19 @@
 from watchdog.observers import polling
 
 
+class NS:
```

Comment:
That would be more brittle. The namespace that returns from the script can contain arbitrary names. What happens if "__getattribute__" is in there? I mean, I could sit and analyse this to figure it out, but why bother if we can do something simple that's safer?